packages/gatsby-theme/src/blocks/ContactBlock/index.js

```diff
@@ -4,8 +4,8 @@ import { Contact } from "ui"
 
 const ContactBlock = ({ heading, align, contactInfo }) => {
   return (
-    <Contact>
-      <Contact.Header align={align}>{heading}</Contact.Header>
+    <Contact align={align}>
```

Comment:
I think the proptypes for the Contact component indicates that a `title` is required, maybe we could change the proptype if the block is intended to be used without a title